language_service/src/test_utils.rs

```diff
@@ -33,11 +33,18 @@ pub(crate) fn compile_with_fake_stdlib(source_name: &str, source_contents: &str)
     let std_source_map = SourceMap::new(
         [(
             "<std>".into(),
-            "namespace FakeStdLib {
+            r#"namespace FakeStdLib {
                 operation Fake() : Unit {}
                 operation FakeWithParam(x: Int) : Unit {}
                 operation FakeCtlAdj() : Unit is Ctl + Adj {}
-            }"
+                newtype Udt = (x: Int, y: Int);
+                newtype UdtWrapper = (inner: Udt);
+                newtype UdtFn = (Int -> Int);
+                newtype UdtFnWithUdtParams = (Udt -> Udt);
+                function TakesUdt(input : Udt) : Udt {
+                    fail "not implemented"
```

Comment:
I wish it could, but for stdlib to compile without errors, I need to have something in there (because of the return type). Unlike user code, stdlib being error-free is a core assumption in the compiler. I thought a `fail` statement would be the most trivial thing to have, but maybe it's less confusing to just return a `Udt`?

Reply:
Oh that makes sense. Thanks

Reply:
Leaving it as is is fine, given the explanation.